### ISSUE TITLE
Custom vibrate patterns

### ISSUE THREAD

**Issue by aaronsilber** [signalapp/Signal-Android#230]
For a long time I have relied on "custom vibrate patterns" to extend the duration and intensity of the vibrating notification. I often can't feel it in my pocket without this. I believe it is stock with AOSP messaging, or it may be a CyanogenMod feature.

**Comment by sbedell** [comment]
Any updates on this?

**Comment by Meteor0id** [comment]
I would like to extent this idea with the possibility of **multiple** custom vibration patterns, so that the user may set a different vibration setting for each conversation. This may have been the intent all along, but it hasn't been clearly stated before in this manner.

**Comment by Meteor0id** [comment]
I would like to add that I noticed another ticked on this same feature request has been closed by the owner with the following argument:
> Thanks @agrajaghh, but I think we're going to skip this for now. It's a lot of code and includes a preference migration, and we're not sure that it's something the majority of users will find useful. We want to focus on customization for the core settings first.

Before the same thing happens to this ticket, I would like to stress the importants of keeping such tickets open nevertheless. It offers a place for people to react to eachothers ideas. Closing a ticket because the feature is 'far out', doesn't mean its too early to think about it, and with thinking, a place to share those thoughts is required. It doesn't need to be implemented this year, but lets keep the idea on standby over here.
